### PR TITLE
PWGHF: fix processMc for sigmac.

### PR DESCRIPTION
**Pull request by mfaggin**

### CODE REVIEW

**Comment by mfaggin** [comment]
@fgrosa @fcolamar @fcatalan92 not sure if maybe it's better to go with [this PR](https://github.com/AliceO2Group/O2Physics/pull/3826) before

**Review by fgrosa — APPROVED** [review]

File: PWGHF/D2H/Tasks/taskSigmac.cxx [hunk 1]
```diff
@@ -881,17 +881,24 @@ struct HfTaskSigmac {
     if (enableTHn) {
       /// loop over Λc+ candidates w/o Σc0,++ mass-window cut
       for (const auto& candidateLc : candidatesLc) {
+        if (!(std::abs(candidateLc.flagMcMatchRec()) == 1 << aod::hf_cand_3prong::DecayType::LcToPKPi)) {
```
Comment (fgrosa) [inline PWGHF/D2H/Tasks/taskSigmac.cxx:884]:
```suggestion
        if (!TESTBIT(std::abs(candidateLc.flagMcMatchRec()), aod::hf_cand_3prong::DecayType::LcToPKPi)) {
```

**Review by fgrosa — CHANGES_REQUESTED** [review]
Ciao @mfaggin I just have a minor comment to improve the readability

**Comment by mfaggin** [comment]
> Ciao @mfaggin I just have a minor comment to improve the readability

thanks @fgrosa , done!

**Review by fgrosa — APPROVED** [review]

**Pull request merged**